adapters/huaweiads/huaweiads_test.go

```diff
@@ -0,0 +1,18 @@
+package huaweiads
+
+import (
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"testing"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderHuaweiAds, config.Adapter{
+		Endpoint: "https://acd.op.hicloud.com/ppsadx/getResult"})
```

Comment:
Consider using an obviously fake endpoint for the purposes of tests. This doesn't need to be the real one here.

Reply:
corrected